mgmt_cli_test.py

```diff
@@ -1359,6 +1360,16 @@ def get_restore_extra_parameters(self) -> str:
         extra_params = self.params.get('mgmt_restore_extra_params')
         return extra_params if extra_params else None
 
+    def _send_restore_results_to_argus(self, task: RestoreTask, manager_version_timestamp: int):
+        restore_time = task.duration.total_seconds()
+        repair_time = task.post_restore_repair_duration.total_seconds()
+        results = {
+            "overall_restore": restore_time,
+            "download_and_load&stream": (restore_time - repair_time),
+            "repair": repair_time,
+        }
```

Comment:
please make it ordered dict - as it might update Argus table metadata randomly due change in columns ordering.
also, consider better naming and order (like: restore time, repair time and total)

Reply:
Didn't get your point about ordered dict as since Python3.7 basic Python dict is ordered by default.